java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1031,6 +1031,34 @@ public void zcard(BaseClient client) {
         assertTrue(executionException.getCause() instanceof RequestException);
     }
 
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void zmscore(BaseClient client) {
+        String key1 = UUID.randomUUID().toString();
+        String key2 = UUID.randomUUID().toString();
+        Map<String, Double> membersScores = Map.of("one", 1.0, "two", 2.0, "three", 3.0);
+        assertEquals(3, client.zadd(key1, membersScores).get());
+        assertArrayEquals(
+                new Double[] {1.0, 2.0, 3.0},
+                client.zmscore(key1, new String[] {"one", "two", "three"}).get());
+        assertArrayEquals(
+                new Double[] {2.0, 3.0}, client.zmscore(key1, new String[] {"two", "three"}).get());
+        assertArrayEquals(
+                new Double[] {1.0, null, 3.0},
+                client.zmscore(key1, new String[] {"one", "nonExistentMember", "three"}).get());
+        assertArrayEquals(
+                new Double[] {null},
+                client.zmscore("nonExistentKey", new String[] {"nonExistentMember"}).get());
```

Comment:
```
        assertArrayEquals(
                new Double[] {null, null},
                client.zmscore("nonExistentKey", new String[] {"nonExistentMember", nonExistentMember}).get());
```
Turns out, you get multiple nulls if you pass multiple non-existent members of a non-existent key...?

Reply:
Yes.